docs/releases/2.2.txt

```diff
@@ -140,6 +140,9 @@ Database backends
 
 * Added result streaming for :meth:`.QuerySet.iterator` on SQLite.
 
+* The Oracle backend client is modified so that the :djadmin:`dbshell`
```

Comment:
Put this under "Management Commands":

"On Oracle, dbshell is wrapped with rlwrap, if available."

It could also be useful to explain the benefit of this (unless it's obvious to Oracle users).